src/shared/template.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Remove leading and trailing empty lines from a string.
+ */
+const trimEmptyLines = (str: string): string =>
+  str.replace(/^\s*\n|\n\s*$/g, '');
```

Comment:
I t should be possible to achieve this by `str.split('\n').filter(line => line.trim().length > 0).join('\n')` if that's more clear.